app/controllers/admins/attachments_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Admins
+  class AttachmentsController < ApplicationController
+    def index
+      @attachments = Attachment.all
+    end
+
+    def show
+      @attachment = Attachment.find(params[:id])
```

Comment:
move in to method and call in `before_action`

Reply:
done